src/main/drivers/timer_def.h

```diff
@@ -1004,4 +1014,41 @@
 #define DEF_TIM_AF__PI6__TCH_TIM8_CH2     D(3, 8)
 #define DEF_TIM_AF__PI7__TCH_TIM8_CH3     D(3, 8)
 
+
+#endif
+
+
+#ifdef USE_BBSHOT
```

Comment:
This seems to be the wrong file for this - it is used to define the MCU hardware. Bitbanging Dshot specific defines should probably go into a file of their own.

Reply:
Yup - this is now in dshot_bitbang.c